### DISCUSSION TITLE
Supports for the UR manipulator family

### DISCUSSION THREAD

**Discussion by ZXP-S-works** [NVlabs/curobo#213] [Software Q&A]
Hi Curobo researchers,

Thanks for the good work! I am curious whether the future release will support the UR family of robots, e.g., UR5, UR10, etc.

Thank you!

Best,
XP

**Comment by balakumar-s** [comment]
Are you referring to the CB3 series? We provide configuration files for UR5e and UR10e. You can also create the configuration for any robot following: https://curobo.org/source/tutorials/1_robot_configuration.html

**Reply by ZXP-S-works** [reply]
Hi Balakumar,

Thanks for the quick reply!

I saw the tutorial that we can manually configure one. It would be great to support these popular robots so that a lot of users don't need to customize their planners repeatedly.

Best,
XP
